plugins/foreman_maintain/0.2/index.md

```diff
@@ -0,0 +1,405 @@
+... LIST SHORTENED ...
+
+Done with backup: 2018-05-03 04:03:41 +0000
+**** BACKUP Complete, contents can be found in: /tmp/my_backup_dir ****
+```
+Note there was no subdir created. 
+
+Also note that when using `--preserve-directory` no data are removed from it when the backup fail.
+
+### Offline backup
+Offline backup is safe way to backup your server for later restoring. All the services are down during the backup and the instance is in a maintenance mode thus not accessible from outside. The instance is started back into normal after the backup finished.
+
+Possible options are:
+```
+# foreman-maintain backup offline -h
+Usage:
+    foreman-maintain backup offline [OPTIONS] BACKUP_DIR
+
+Parameters:
+    BACKUP_DIR                    Path to backup dir
+
+Options:
+    -y, --assumeyes               Automatically answer yes for all questions
+    -w, --whitelist whitelist     Comma-separated list of labels of steps to be skipped
+    -f, --force                   Force steps that would be skipped as they were already run
+    -s, --skip-pulp-content       Do not backup Pulp content
+    -p, --preserve-directory      Do not create a time-stamped subdirectory
+    -t, --split-pulp-tar SPLIT_SIZE Split pulp data into files of a specified size, i.e. (100M, 50G). See '--tape-length' in 'info tar' for all sizes
+    -i, --incremental PREVIOUS_BACKUP_DIR Backup changes since previous backup
+    --features FEATURES           Foreman Proxy features to include in the backup. Valid features are tftp, dns, dhcp, openscap, and all. (comma-separated list)
+    --include-db-dumps            Also dump full database schema before offline backup
+    -h, --help                    print help
+```
+As the archives of Pulp data can be large it may be useful to split it into volumes using `--split-pulp-tar` options. 
+
+There is also `--include-db-dumps` option to take extra dumps of the DBs and store them with the backup. Note the dumps are done with all the service running so the data integrity among DBs is not ensured. If you want to rely on the dumps during restore, make sure all the services but local DBs are down prior running
+the backup.
+
+### Online backup
+The online backup is intended for making a copy of the data for debugging purposes only.
+The backup routine can not ensure 100% consistency while the backup is taking place as there is a chance there may be data mismatch between Mongo and Postgres databases while the services are live.
+
+If you wish to utilize the online backup for production use you need to ensure that there are no modifications occurring during your backup run.
+
+During this backup a dump of the underlying databases (PgSQL, Mongo) is done. While Pulp repositories are archived we check for a changes in the data. When something changed the backup is re-run automatically so make sure there no repo syncs, promotions or other changes to the Pulp content performed during the backup. 
```

Comment:
`for changes in the data1

Reply:
`When something changes, the backup is run-run automatically so make sure there are no`.

Additionally may add:

"We recommend treating the server as read-only during an online backup to".

Also, it's probably worth mentioning explicitly that the re-run automatically can mean that an online backup will take hours to days to complete or never complete at all if changes happen faster than the backup.